index.bs

```diff
@@ -514,6 +514,19 @@ See also:
 
 This section details design principles for features which are exposed via CSS.
 
+<h3 id="css-content-should-be-visible">Content should be readable by default</h3>
+
+Design CSS properties or CSS layout systems (which are typically values of the `display` property), 
```

Comment:
```suggestion
Design CSS properties or CSS layout systems 
```

I'm not sure we need the parenthetical in this summary sentence. I think we can include it a little later on.